src/components/UI/SvgIcon/SvgIcon.stories.mdx

```diff
@@ -77,7 +89,7 @@ Icons used in application.
 
 <Preview>
   <Story name="TooltipIcon">
-    <TooltipIcon cssClass="feather-info" content={<>dummy info here</>}>
```

Comment:
It's not clear in he storybook that this part is about displaying a tooltip, because the title is the same as the previous section